### ISSUE TITLE
Add comparisons of INR, Sensex and other economical parameters between previous and current government

### ISSUE THREAD

**Issue by bogas04** [reddit-india/indianelectionmeter.github.io#52]
Need to query some API for the same and display results in the <App />, not hard to produce.

**Comment by PCMR-FSNG** [comment]
Why do we need a comparison mechanism for tracking of promises ?

**Comment by kritinvij** [comment]
I think this would just help us keep track of how the current govt is doing as compared to the previous one. I vote this goes in v2/v3.

**Comment by satyarth** [comment]
TBH, this seems like an entirely separate thing to tracking promises. Also, just comparing economic data like that doesn't really tell you much in my opinion.

**Comment by okpatil4u** [comment]
I agree with @satyarth, value of rupee is a strategic, have a look at china, they have been depreciating their currency for a while. But we could track other indices, like hdi or ease of doing business. This will prove useful.

**Comment by srinarasi** [comment]
+1 to what @satyarth said. This project should be about tracking promises.

**Comment by greyhound291** [comment]
We could use the API's to track the vague promises like bring down coal imports, or increasing literacy rates etc.

Should I make a seperate issue for this?

**Comment by bogas04** [comment]
Yeah the project is about promises but you often find tweets comparing prices of petrol, INR, coal and what not between governments. Ultimate goal is to track the progress made by new government, this can help a tad bit more to see real life changes. INR itself is meaning less but other prices together make some sense.
